components/dropdown/dropdown-content-styles.js

```diff
@@ -192,6 +194,12 @@ export const dropdownContentStyles = css`
 		}
 	}
 
+	@media (prefers-reduced-motion: reduce) {
+		:host([close-radio]) {
+			animation: var(--d2l-dropdown-close-animation-name) 300ms ease-out;
+		}
+	}
+
```

Comment:
`prefers-reduced-motion: reduce` actually means that someone would not want to see the animation, rather than they would want to see it. In this case what your code would be doing is that the animation would only be seen by someone who wants that reduced motion.

Another example of this being used in the file is [here](https://github.com/BrightspaceUI/core/blob/5f94fe7938ca697207bdcb0027fd3d8a2d6257b1/components/dropdown/dropdown-content-styles.js#L191) where the animations that were previously set are disabled for the `prefers-reduced-motion` case.

Given that you've moved `close-radio` to be local to `dropdown-menu`, these animation styles that are specific to `dropdown-menu` could also move in there (not sure if others on Gaudi have thoughts on that though - perhaps we do just want all dropdown-content related styles in this file)